Sources/StreamChatUI/CommonViews/CellSeperatorView/CellSeparatorCollectionViewLayout.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+public protocol CellSeparatorContainingLayout {
+    /// The reuse identifier of the cell separator view.
+    var separatorIdentifier: String { get }
+}
+
+public protocol CellSeparatorContainingLayoutDelegate {
+    /// Implement this method to have detailed control over the visibility of the cell separators.
+    func collectionView(
+        _ collectionView: UICollectionView,
+        layout collectionViewLayout: CellSeparatorContainingLayout & UICollectionViewFlowLayout,
+        shouldShowSeparatorForCellAtIndexPath indexPath: IndexPath
+    ) -> Bool
+}
+
+// Default implementation of CellSeparatorContainingLayoutDelegate
+
+public extension CellSeparatorContainingLayoutDelegate {
+    func collectionView(
+        _ collectionView: UICollectionView,
+        layout collectionViewLayout: CellSeparatorContainingLayout & UICollectionViewFlowLayout,
+        shouldShowSeparatorForCellAtIndexPath indexPath: IndexPath
+    ) -> Bool { true }
+}
```

Comment:
The delegate makes sense, but the other approach would make it possible to change this through UIConfig. But make we don't want that? 🤔